decred/examples/create_testnet_wallet.py

```diff
@@ -16,15 +16,20 @@
 from decred.wallet.wallet import Wallet
 
 
-# Create an encrypted, password-protected wallet file.
-password = getpass()
-mkdir("testnet")
-walletPath = os.path.join("testnet", "testnet_wallet.db")
-mnemonicSeed, wallet = Wallet.create(walletPath, password)
-# Open the zeroth Decred account to get an address.
-wallet.open("dcr", 0, password, None)
-
-# Print the seed words and an address.
-print("Mnemonic seed\n-------------")
-print(" ".join(mnemonicSeed))
-print("Receive DCR at %s" % wallet.currentAddress())
+def run():
+    # Create an encrypted, password-protected wallet file.
+    password = getpass()
+    mkdir("testnet")
+    walletPath = os.path.join("testnet", "testnet_wallet.db")
+    mnemonicSeed, wallet = Wallet.create(walletPath, password)
+    # Open the zeroth Decred account to get an address.
+    wallet.open("dcr", 0, password, None)
+
+    # Print the seed words and an address.
+    print("Mnemonic seed\n-------------")
+    print(" ".join(mnemonicSeed))
+    print("Receive DCR at %s" % wallet.currentAddress())
+
+
+if __name__ == "__main__":
+    run()
```

Comment:
It seems common to call this `main`, unless there's some reason to call it differently. Same goes for the other scripts.